msal/src/main/java/com/microsoft/identity/client/internal/controllers/LocalMSALController.java

```diff
@@ -172,22 +175,14 @@ public AcquireTokenResult acquireTokenSilent(
                 "Acquiring token silently..."
         );
 
-        final List<String> msalScopes = new ArrayList<>();
-        msalScopes.add("openid");
-        msalScopes.add("profile");
-        msalScopes.add("offline_access");
-
-        if (!parameters.getScopes().containsAll(msalScopes)) {
-            parameters.getScopes().addAll(msalScopes);
-            // Sanitize-out any null or empty scopes
-            parameters.getScopes().removeAll(Arrays.asList("", null));
-        }
-
         final AcquireTokenResult acquireTokenSilentResult = new AcquireTokenResult();
 
         //Validate MSAL Parameters
         parameters.validate();
 
+        // Add default scopes
+        addDefaultScopes(parameters);
```

Comment:
If the default scopes string is required, would it be better to change the order here and add the scope validation inside `parameters.validate();`?

Reply:
Not really right, that's the validation for input request parameters. Also if we did this we would need to make changes for scope validation in base class as well where we check if scopes are not empty (User provided atleast one) but adding these would break that behavior